pages/contribute/style_guide.md

```diff
@@ -50,9 +50,40 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
   * **Training:** training is an uncountable noun and cannot have a plural. You can write "training courses" and "training materials" but not "trainings".
 
 ## Bibliography
- * Add your citations as bibtex to the `_bibliography/references.bib` file.
- * Add {% raw %} `{% cite reference_key %}` {% endraw %} to the text where you are citing the reference.
- * Add {% raw %} `## Bibliography {% bibliography --cited %}` {% endraw %} to show a bibliography section containing the references in the page.
+ * RDMkit uses BibTeX as a tool for creating bibliography.
+ * Follow the following instuctions to create and display a bibiography on your page:
+   1. Add your citations in the `_bibliography/references.bib` file. Refer to the instructions below for more information on the format.
+   1. Add {% raw %} `{% cite reference_key %}` {% endraw %} to the text where you citing one of your entries.
+   1. Add {% raw %} `## Bibliography {% bibliography --cited %}` {% endraw %} in your page to show a bibliography section containing the references you have added following the instructions in the previous point. 
+* We use a display class based on *Nature* publications. Your BibTeX files should follow a standardised format described below.
+  We recommend using {% tool "zotero" %} to automatise the process (see below).
+* An example of the minimal structure required for BibTeX entries is:
+
+```
+﻿@article{surname12024lorem,
+  title = {Lorem Ipsum},
+  author = {Surname1, Alice and Surname2, Bob}
+  journal = {Journal},
+  volume = {1},
+  pages = {1--10},
+  year = {2024},
+  number = {1},
+  doi = {DOI},
+  langid = {english}
+}
+```
+Make sure that the *author* field does not contain *et al.*, as this substitution is performed via the class and should
+not be part of the information provided by the contributors.
+
+* If you are a {% tool "zotero" %} user, we recommed
+  1. Install the {% tool "better-bibtex" %} plugin.
+  1. On your desktop version of {% tool "zotero" %}, open the *Better BibTeX preferences* (under *Tools > Better BibTeX*).
```

Comment:
```suggestion
  2. On your desktop version of {% tool "zotero" %}, open the *Better BibTeX preferences* (under *Tools > Better BibTeX*).
```
All the numbers are "1." - is that intentional?

Reply:
That's what we recommend to do [here](https://github.com/elixir-europe/rdmkit/blob/a716351446f11993989b3d948cde78714decb8c4/pages/contribute/markdown_cheat_sheet.md), I think.